{
    "version": "https://jsonfeed.org/version/1",
    "title": "Security Advisory for Python packages hosted at PyPI.org",
    "home_page_url": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip",
    "feed_url": "https://azu.github.io/github-advisory-database-rss/pip.json",
    "description": "Security Advisory for Python packages hosted at PyPI.org on GitHub",
    "items": [
        {
            "content_html": "<p>The (1) <code>contrib.sessions.backends.base.SessionBase.flush</code> and (2) <code>cache_db.SessionStore.flush</code> functions in Django 1.7.x before 1.7.10, 1.4.x before 1.4.22, and possibly other versions create empty sessions in certain circumstances, which allows remote attackers to cause a denial of service (session store consumption) via unspecified vectors.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2015-5964\">https://nvd.nist.gov/vuln/detail/CVE-2015-5964</a></li>\n<li><a href=\"http://lists.fedoraproject.org/pipermail/package-announce/2015-November/172084.html\">http://lists.fedoraproject.org/pipermail/package-announce/2015-November/172084.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2015-1766.html\">http://rhn.redhat.com/errata/RHSA-2015-1766.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2015-1767.html\">http://rhn.redhat.com/errata/RHSA-2015-1767.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2015-1894.html\">http://rhn.redhat.com/errata/RHSA-2015-1894.html</a></li>\n<li><a href=\"http://www.debian.org/security/2015/dsa-3338\">http://www.debian.org/security/2015/dsa-3338</a></li>\n<li><a href=\"http://www.oracle.com/technetwork/topics/security/bulletinoct2015-2511968.html\">http://www.oracle.com/technetwork/topics/security/bulletinoct2015-2511968.html</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-2720-1\">http://www.ubuntu.com/usn/USN-2720-1</a></li>\n<li><a href=\"https://web.archive.org/web/20200228051814/http://www.securityfocus.com/bid/76440\">https://web.archive.org/web/20200228051814/http://www.securityfocus.com/bid/76440</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2015/aug/18/security-releases\">https://www.djangoproject.com/weblog/2015/aug/18/security-releases</a></li>\n<li><a href=\"https://github.com/django/django/commit/2f5485346ee6f84b4e52068c04e043092daf55f7\">https://github.com/django/django/commit/2f5485346ee6f84b4e52068c04e043092daf55f7</a></li>\n<li><a href=\"https://github.com/django/django/commit/575f59f9bc7c59a5e41a081d1f5f55fc859c5012\">https://github.com/django/django/commit/575f59f9bc7c59a5e41a081d1f5f55fc859c5012</a></li>\n<li><a href=\"https://github.com/django/django/commit/8cc41ce7a7a8f6bebfdd89d5ab276cd0109f4fc5\">https://github.com/django/django/commit/8cc41ce7a7a8f6bebfdd89d5ab276cd0109f4fc5</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2015-23.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2015-23.yaml</a></li>\n<li><a href=\"https://web.archive.org/web/20150906124157/http://www.securityfocus.com/bid/76440\">https://web.archive.org/web/20150906124157/http://www.securityfocus.com/bid/76440</a></li>\n<li><a href=\"https://web.archive.org/web/20150923161048/http://www.securitytracker.com/id/1033318\">https://web.archive.org/web/20150923161048/http://www.securitytracker.com/id/1033318</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-x38m-486c-2wr9\">https://github.com/advisories/GHSA-x38m-486c-2wr9</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-x38m-486c-2wr9",
            "title": "[Django] Denial-of-service possibility in logout() view by filling session store",
            "date_modified": "2024-09-18T19:59:53.000Z",
            "date_published": "2022-05-17T03:16:12.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-x38m-486c-2wr9"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>The (1) <code>contrib.sessions.backends.base.SessionBase.flush</code> and (2) <code>cache_db.SessionStore.flush</code> functions in Django 1.7.x before 1.7.10, 1.4.x before 1.4.22, and possibly other versions create empty sessions in certain circumstances, which allows remote attackers to cause a denial of service (session store consumption) via unspecified vectors.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2015-5964\">https://nvd.nist.gov/vuln/detail/CVE-2015-5964</a></li>\n<li><a href=\"http://lists.fedoraproject.org/pipermail/package-announce/2015-November/172084.html\">http://lists.fedoraproject.org/pipermail/package-announce/2015-November/172084.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2015-1766.html\">http://rhn.redhat.com/errata/RHSA-2015-1766.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2015-1767.html\">http://rhn.redhat.com/errata/RHSA-2015-1767.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2015-1894.html\">http://rhn.redhat.com/errata/RHSA-2015-1894.html</a></li>\n<li><a href=\"http://www.debian.org/security/2015/dsa-3338\">http://www.debian.org/security/2015/dsa-3338</a></li>\n<li><a href=\"http://www.oracle.com/technetwork/topics/security/bulletinoct2015-2511968.html\">http://www.oracle.com/technetwork/topics/security/bulletinoct2015-2511968.html</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-2720-1\">http://www.ubuntu.com/usn/USN-2720-1</a></li>\n<li><a href=\"https://web.archive.org/web/20200228051814/http://www.securityfocus.com/bid/76440\">https://web.archive.org/web/20200228051814/http://www.securityfocus.com/bid/76440</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2015/aug/18/security-releases\">https://www.djangoproject.com/weblog/2015/aug/18/security-releases</a></li>\n<li><a href=\"https://github.com/django/django/commit/2f5485346ee6f84b4e52068c04e043092daf55f7\">https://github.com/django/django/commit/2f5485346ee6f84b4e52068c04e043092daf55f7</a></li>\n<li><a href=\"https://github.com/django/django/commit/575f59f9bc7c59a5e41a081d1f5f55fc859c5012\">https://github.com/django/django/commit/575f59f9bc7c59a5e41a081d1f5f55fc859c5012</a></li>\n<li><a href=\"https://github.com/django/django/commit/8cc41ce7a7a8f6bebfdd89d5ab276cd0109f4fc5\">https://github.com/django/django/commit/8cc41ce7a7a8f6bebfdd89d5ab276cd0109f4fc5</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2015-23.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2015-23.yaml</a></li>\n<li><a href=\"https://web.archive.org/web/20150906124157/http://www.securityfocus.com/bid/76440\">https://web.archive.org/web/20150906124157/http://www.securityfocus.com/bid/76440</a></li>\n<li><a href=\"https://web.archive.org/web/20150923161048/http://www.securitytracker.com/id/1033318\">https://web.archive.org/web/20150923161048/http://www.securitytracker.com/id/1033318</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-x38m-486c-2wr9\">https://github.com/advisories/GHSA-x38m-486c-2wr9</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-x38m-486c-2wr9",
            "title": "[Django] Denial-of-service possibility in logout() view by filling session store",
            "date_modified": "2024-09-18T19:59:53.000Z",
            "date_published": "2022-05-17T03:16:12.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-x38m-486c-2wr9"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>The <code>django.util.http.is_safe_url</code> function in Django before 1.4.18, 1.6.x before 1.6.10, and 1.7.x before 1.7.3 does not properly handle leading whitespaces, which allows remote attackers to conduct cross-site scripting (XSS) attacks via a crafted URL, related to redirect URLs, as demonstrated by a <code>\\njavascript:</code> URL.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2015-0220\">https://nvd.nist.gov/vuln/detail/CVE-2015-0220</a></li>\n<li><a href=\"http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148485.html\">http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148485.html</a></li>\n<li><a href=\"http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148608.html\">http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148608.html</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2015-04/msg00001.html\">http://lists.opensuse.org/opensuse-updates/2015-04/msg00001.html</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2015-09/msg00035.html\">http://lists.opensuse.org/opensuse-updates/2015-09/msg00035.html</a></li>\n<li><a href=\"http://ubuntu.com/usn/usn-2469-1\">http://ubuntu.com/usn/usn-2469-1</a></li>\n<li><a href=\"https://github.com/django/django/commit/4c241f1b710da6419d9dca160e80b23b82db7758\">https://github.com/django/django/commit/4c241f1b710da6419d9dca160e80b23b82db7758</a></li>\n<li><a href=\"https://github.com/django/django/commit/72e0b033662faa11bb7f516f18a132728aa0ae28\">https://github.com/django/django/commit/72e0b033662faa11bb7f516f18a132728aa0ae28</a></li>\n<li><a href=\"https://github.com/django/django/commit/de67dedc771ad2edec15c1d00c083a1a084e1e89\">https://github.com/django/django/commit/de67dedc771ad2edec15c1d00c083a1a084e1e89</a></li>\n<li><a href=\"https://github.com/django/django/blob/4555a823fd57e261e1b19c778429473256c8ea08/docs/releases/1.4.18.txt#L34-L46\">https://github.com/django/django/blob/4555a823fd57e261e1b19c778429473256c8ea08/docs/releases/1.4.18.txt#L34-L46</a></li>\n<li><a href=\"https://web.archive.org/web/20150128111656/http://secunia.com/advisories/62285\">https://web.archive.org/web/20150128111656/http://secunia.com/advisories/62285</a></li>\n<li><a href=\"https://web.archive.org/web/20150523054951/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:109/?name=MDVSA-2015:109\">https://web.archive.org/web/20150523054951/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:109/?name=MDVSA-2015:109</a></li>\n<li><a href=\"https://web.archive.org/web/20150523054953/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:036/?name=MDVSA-2015:036\">https://web.archive.org/web/20150523054953/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:036/?name=MDVSA-2015:036</a></li>\n<li><a href=\"https://web.archive.org/web/20151104201446/http://secunia.com/advisories/62718\">https://web.archive.org/web/20151104201446/http://secunia.com/advisories/62718</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2015-5.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2015-5.yaml</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2015/jan/13/security\">https://www.djangoproject.com/weblog/2015/jan/13/security</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-gv98-g628-m9x5\">https://github.com/advisories/GHSA-gv98-g628-m9x5</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-gv98-g628-m9x5",
            "title": "[Django] Django Cross-site Scripting Vulnerability",
            "date_modified": "2024-09-18T19:55:31.000Z",
            "date_published": "2022-05-17T03:20:49.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-gv98-g628-m9x5"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>The <code>django.util.http.is_safe_url</code> function in Django before 1.4.18, 1.6.x before 1.6.10, and 1.7.x before 1.7.3 does not properly handle leading whitespaces, which allows remote attackers to conduct cross-site scripting (XSS) attacks via a crafted URL, related to redirect URLs, as demonstrated by a <code>\\njavascript:</code> URL.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2015-0220\">https://nvd.nist.gov/vuln/detail/CVE-2015-0220</a></li>\n<li><a href=\"http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148485.html\">http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148485.html</a></li>\n<li><a href=\"http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148608.html\">http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148608.html</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2015-04/msg00001.html\">http://lists.opensuse.org/opensuse-updates/2015-04/msg00001.html</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2015-09/msg00035.html\">http://lists.opensuse.org/opensuse-updates/2015-09/msg00035.html</a></li>\n<li><a href=\"http://ubuntu.com/usn/usn-2469-1\">http://ubuntu.com/usn/usn-2469-1</a></li>\n<li><a href=\"https://github.com/django/django/commit/4c241f1b710da6419d9dca160e80b23b82db7758\">https://github.com/django/django/commit/4c241f1b710da6419d9dca160e80b23b82db7758</a></li>\n<li><a href=\"https://github.com/django/django/commit/72e0b033662faa11bb7f516f18a132728aa0ae28\">https://github.com/django/django/commit/72e0b033662faa11bb7f516f18a132728aa0ae28</a></li>\n<li><a href=\"https://github.com/django/django/commit/de67dedc771ad2edec15c1d00c083a1a084e1e89\">https://github.com/django/django/commit/de67dedc771ad2edec15c1d00c083a1a084e1e89</a></li>\n<li><a href=\"https://github.com/django/django/blob/4555a823fd57e261e1b19c778429473256c8ea08/docs/releases/1.4.18.txt#L34-L46\">https://github.com/django/django/blob/4555a823fd57e261e1b19c778429473256c8ea08/docs/releases/1.4.18.txt#L34-L46</a></li>\n<li><a href=\"https://web.archive.org/web/20150128111656/http://secunia.com/advisories/62285\">https://web.archive.org/web/20150128111656/http://secunia.com/advisories/62285</a></li>\n<li><a href=\"https://web.archive.org/web/20150523054951/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:109/?name=MDVSA-2015:109\">https://web.archive.org/web/20150523054951/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:109/?name=MDVSA-2015:109</a></li>\n<li><a href=\"https://web.archive.org/web/20150523054953/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:036/?name=MDVSA-2015:036\">https://web.archive.org/web/20150523054953/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:036/?name=MDVSA-2015:036</a></li>\n<li><a href=\"https://web.archive.org/web/20151104201446/http://secunia.com/advisories/62718\">https://web.archive.org/web/20151104201446/http://secunia.com/advisories/62718</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2015-5.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2015-5.yaml</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2015/jan/13/security\">https://www.djangoproject.com/weblog/2015/jan/13/security</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-gv98-g628-m9x5\">https://github.com/advisories/GHSA-gv98-g628-m9x5</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-gv98-g628-m9x5",
            "title": "[Django] Django Cross-site Scripting Vulnerability",
            "date_modified": "2024-09-18T19:55:31.000Z",
            "date_published": "2022-05-17T03:20:49.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-gv98-g628-m9x5"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>The <code>django.util.http.is_safe_url</code> function in Django before 1.4.18, 1.6.x before 1.6.10, and 1.7.x before 1.7.3 does not properly handle leading whitespaces, which allows remote attackers to conduct cross-site scripting (XSS) attacks via a crafted URL, related to redirect URLs, as demonstrated by a <code>\\njavascript:</code> URL.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2015-0220\">https://nvd.nist.gov/vuln/detail/CVE-2015-0220</a></li>\n<li><a href=\"http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148485.html\">http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148485.html</a></li>\n<li><a href=\"http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148608.html\">http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148608.html</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2015-04/msg00001.html\">http://lists.opensuse.org/opensuse-updates/2015-04/msg00001.html</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2015-09/msg00035.html\">http://lists.opensuse.org/opensuse-updates/2015-09/msg00035.html</a></li>\n<li><a href=\"http://ubuntu.com/usn/usn-2469-1\">http://ubuntu.com/usn/usn-2469-1</a></li>\n<li><a href=\"https://github.com/django/django/commit/4c241f1b710da6419d9dca160e80b23b82db7758\">https://github.com/django/django/commit/4c241f1b710da6419d9dca160e80b23b82db7758</a></li>\n<li><a href=\"https://github.com/django/django/commit/72e0b033662faa11bb7f516f18a132728aa0ae28\">https://github.com/django/django/commit/72e0b033662faa11bb7f516f18a132728aa0ae28</a></li>\n<li><a href=\"https://github.com/django/django/commit/de67dedc771ad2edec15c1d00c083a1a084e1e89\">https://github.com/django/django/commit/de67dedc771ad2edec15c1d00c083a1a084e1e89</a></li>\n<li><a href=\"https://github.com/django/django/blob/4555a823fd57e261e1b19c778429473256c8ea08/docs/releases/1.4.18.txt#L34-L46\">https://github.com/django/django/blob/4555a823fd57e261e1b19c778429473256c8ea08/docs/releases/1.4.18.txt#L34-L46</a></li>\n<li><a href=\"https://web.archive.org/web/20150128111656/http://secunia.com/advisories/62285\">https://web.archive.org/web/20150128111656/http://secunia.com/advisories/62285</a></li>\n<li><a href=\"https://web.archive.org/web/20150523054951/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:109/?name=MDVSA-2015:109\">https://web.archive.org/web/20150523054951/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:109/?name=MDVSA-2015:109</a></li>\n<li><a href=\"https://web.archive.org/web/20150523054953/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:036/?name=MDVSA-2015:036\">https://web.archive.org/web/20150523054953/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:036/?name=MDVSA-2015:036</a></li>\n<li><a href=\"https://web.archive.org/web/20151104201446/http://secunia.com/advisories/62718\">https://web.archive.org/web/20151104201446/http://secunia.com/advisories/62718</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2015-5.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2015-5.yaml</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2015/jan/13/security\">https://www.djangoproject.com/weblog/2015/jan/13/security</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-gv98-g628-m9x5\">https://github.com/advisories/GHSA-gv98-g628-m9x5</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-gv98-g628-m9x5",
            "title": "[django] Django Cross-site Scripting Vulnerability",
            "date_modified": "2024-09-18T19:55:31.000Z",
            "date_published": "2022-05-17T03:20:49.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-gv98-g628-m9x5"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>The <code>django.views.static.serve</code> view in Django before 1.4.18, 1.6.x before 1.6.10, and 1.7.x before 1.7.3 reads files an entire line at a time, which allows remote attackers to cause a denial of service (memory consumption) via a long line in a file.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2015-0221\">https://nvd.nist.gov/vuln/detail/CVE-2015-0221</a></li>\n<li><a href=\"http://advisories.mageia.org/MGASA-2015-0026.html\">http://advisories.mageia.org/MGASA-2015-0026.html</a></li>\n<li><a href=\"http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148485.html\">http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148485.html</a></li>\n<li><a href=\"http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148608.html\">http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148608.html</a></li>\n<li><a href=\"http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148696.html\">http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148696.html</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2015-04/msg00001.html\">http://lists.opensuse.org/opensuse-updates/2015-04/msg00001.html</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2015-09/msg00035.html\">http://lists.opensuse.org/opensuse-updates/2015-09/msg00035.html</a></li>\n<li><a href=\"http://ubuntu.com/usn/usn-2469-1\">http://ubuntu.com/usn/usn-2469-1</a></li>\n<li><a href=\"https://github.com/django/django/commit/553779c4055e8742cc832ed525b9ee34b174934f\">https://github.com/django/django/commit/553779c4055e8742cc832ed525b9ee34b174934f</a></li>\n<li><a href=\"https://github.com/django/django/commit/818e59a3f0fbadf6c447754d202d88df025f8f2a\">https://github.com/django/django/commit/818e59a3f0fbadf6c447754d202d88df025f8f2a</a></li>\n<li><a href=\"https://github.com/django/django/commit/d020da6646c5142bc092247d218a3d1ce3e993f7\">https://github.com/django/django/commit/d020da6646c5142bc092247d218a3d1ce3e993f7</a></li>\n<li><a href=\"https://github.com/django/django/blob/9b9c805cedb08621bd5dc58a01a6478eb7cc49a9/docs/releases/1.4.18.txt#L48C1-L49C1\">https://github.com/django/django/blob/9b9c805cedb08621bd5dc58a01a6478eb7cc49a9/docs/releases/1.4.18.txt#L48C1-L49C1</a></li>\n<li><a href=\"https://web.archive.org/web/20150128111656/http://secunia.com/advisories/62285\">https://web.archive.org/web/20150128111656/http://secunia.com/advisories/62285</a></li>\n<li><a href=\"https://web.archive.org/web/20150128111656/http://secunia.com/advisories/62309\">https://web.archive.org/web/20150128111656/http://secunia.com/advisories/62309</a></li>\n<li><a href=\"https://web.archive.org/web/20150523054951/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:109/?name=MDVSA-2015:109\">https://web.archive.org/web/20150523054951/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:109/?name=MDVSA-2015:109</a></li>\n<li><a href=\"https://web.archive.org/web/20150523054953/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:036/?name=MDVSA-2015:036\">https://web.archive.org/web/20150523054953/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:036/?name=MDVSA-2015:036</a></li>\n<li><a href=\"https://web.archive.org/web/20151104201446/http://secunia.com/advisories/62718\">https://web.archive.org/web/20151104201446/http://secunia.com/advisories/62718</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2015/jan/13/security\">https://www.djangoproject.com/weblog/2015/jan/13/security</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2015-6.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2015-6.yaml</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-jhjg-w2cp-5j44\">https://github.com/advisories/GHSA-jhjg-w2cp-5j44</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-jhjg-w2cp-5j44",
            "title": "[Django] Django DoS in django.views.static.serve",
            "date_modified": "2024-09-18T19:54:00.000Z",
            "date_published": "2022-05-17T03:20:50.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-jhjg-w2cp-5j44"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>The <code>django.views.static.serve</code> view in Django before 1.4.18, 1.6.x before 1.6.10, and 1.7.x before 1.7.3 reads files an entire line at a time, which allows remote attackers to cause a denial of service (memory consumption) via a long line in a file.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2015-0221\">https://nvd.nist.gov/vuln/detail/CVE-2015-0221</a></li>\n<li><a href=\"http://advisories.mageia.org/MGASA-2015-0026.html\">http://advisories.mageia.org/MGASA-2015-0026.html</a></li>\n<li><a href=\"http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148485.html\">http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148485.html</a></li>\n<li><a href=\"http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148608.html\">http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148608.html</a></li>\n<li><a href=\"http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148696.html\">http://lists.fedoraproject.org/pipermail/package-announce/2015-January/148696.html</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2015-04/msg00001.html\">http://lists.opensuse.org/opensuse-updates/2015-04/msg00001.html</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2015-09/msg00035.html\">http://lists.opensuse.org/opensuse-updates/2015-09/msg00035.html</a></li>\n<li><a href=\"http://ubuntu.com/usn/usn-2469-1\">http://ubuntu.com/usn/usn-2469-1</a></li>\n<li><a href=\"https://github.com/django/django/commit/553779c4055e8742cc832ed525b9ee34b174934f\">https://github.com/django/django/commit/553779c4055e8742cc832ed525b9ee34b174934f</a></li>\n<li><a href=\"https://github.com/django/django/commit/818e59a3f0fbadf6c447754d202d88df025f8f2a\">https://github.com/django/django/commit/818e59a3f0fbadf6c447754d202d88df025f8f2a</a></li>\n<li><a href=\"https://github.com/django/django/commit/d020da6646c5142bc092247d218a3d1ce3e993f7\">https://github.com/django/django/commit/d020da6646c5142bc092247d218a3d1ce3e993f7</a></li>\n<li><a href=\"https://github.com/django/django/blob/9b9c805cedb08621bd5dc58a01a6478eb7cc49a9/docs/releases/1.4.18.txt#L48C1-L49C1\">https://github.com/django/django/blob/9b9c805cedb08621bd5dc58a01a6478eb7cc49a9/docs/releases/1.4.18.txt#L48C1-L49C1</a></li>\n<li><a href=\"https://web.archive.org/web/20150128111656/http://secunia.com/advisories/62285\">https://web.archive.org/web/20150128111656/http://secunia.com/advisories/62285</a></li>\n<li><a href=\"https://web.archive.org/web/20150128111656/http://secunia.com/advisories/62309\">https://web.archive.org/web/20150128111656/http://secunia.com/advisories/62309</a></li>\n<li><a href=\"https://web.archive.org/web/20150523054951/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:109/?name=MDVSA-2015:109\">https://web.archive.org/web/20150523054951/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:109/?name=MDVSA-2015:109</a></li>\n<li><a href=\"https://web.archive.org/web/20150523054953/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:036/?name=MDVSA-2015:036\">https://web.archive.org/web/20150523054953/http://www.mandriva.com/en/support/security/advisories/advisory/MDVSA-2015:036/?name=MDVSA-2015:036</a></li>\n<li><a href=\"https://web.archive.org/web/20151104201446/http://secunia.com/advisories/62718\">https://web.archive.org/web/20151104201446/http://secunia.com/advisories/62718</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2015/jan/13/security\">https://www.djangoproject.com/weblog/2015/jan/13/security</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2015-6.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2015-6.yaml</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-jhjg-w2cp-5j44\">https://github.com/advisories/GHSA-jhjg-w2cp-5j44</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-jhjg-w2cp-5j44",
            "title": "[Django] Django DoS in django.views.static.serve",
            "date_modified": "2024-09-18T19:54:00.000Z",
            "date_published": "2022-05-17T03:20:50.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-jhjg-w2cp-5j44"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Django 1.4 before 1.4.13, 1.5 before 1.5.8, 1.6 before 1.6.5, and 1.7 before 1.7b4 does not properly include the (1) Vary: Cookie or (2) Cache-Control header in responses, which allows remote attackers to obtain sensitive information or poison the cache via a request from certain browsers.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2014-1418\">https://nvd.nist.gov/vuln/detail/CVE-2014-1418</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html\">http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html</a></li>\n<li><a href=\"http://www.debian.org/security/2014/dsa-2934\">http://www.debian.org/security/2014/dsa-2934</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2014/05/14/10\">http://www.openwall.com/lists/oss-security/2014/05/14/10</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2014/05/15/3\">http://www.openwall.com/lists/oss-security/2014/05/15/3</a></li>\n<li><a href=\"https://github.com/django/django/commit/1abcf3a808b35abae5d425ed4d44cb6e886dc769\">https://github.com/django/django/commit/1abcf3a808b35abae5d425ed4d44cb6e886dc769</a></li>\n<li><a href=\"https://github.com/django/django/commit/28e23306aa53bbbb8fb87db85f99d970b051026c\">https://github.com/django/django/commit/28e23306aa53bbbb8fb87db85f99d970b051026c</a></li>\n<li><a href=\"https://github.com/django/django/commit/4001ec8698f577b973c5a540801d8a0bbea1205b\">https://github.com/django/django/commit/4001ec8698f577b973c5a540801d8a0bbea1205b</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued\">https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2014-19.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2014-19.yaml</a></li>\n<li><a href=\"http://ubuntu.com/usn/usn-2212-1\">http://ubuntu.com/usn/usn-2212-1</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-q7q2-qf2q-rw3w\">https://github.com/advisories/GHSA-q7q2-qf2q-rw3w</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-q7q2-qf2q-rw3w",
            "title": "[Django] Django Vulnerable to Cache Poisoning",
            "date_modified": "2024-09-18T19:51:16.000Z",
            "date_published": "2022-05-17T03:07:00.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-q7q2-qf2q-rw3w"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Django 1.4 before 1.4.13, 1.5 before 1.5.8, 1.6 before 1.6.5, and 1.7 before 1.7b4 does not properly include the (1) Vary: Cookie or (2) Cache-Control header in responses, which allows remote attackers to obtain sensitive information or poison the cache via a request from certain browsers.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2014-1418\">https://nvd.nist.gov/vuln/detail/CVE-2014-1418</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html\">http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html</a></li>\n<li><a href=\"http://www.debian.org/security/2014/dsa-2934\">http://www.debian.org/security/2014/dsa-2934</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2014/05/14/10\">http://www.openwall.com/lists/oss-security/2014/05/14/10</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2014/05/15/3\">http://www.openwall.com/lists/oss-security/2014/05/15/3</a></li>\n<li><a href=\"https://github.com/django/django/commit/1abcf3a808b35abae5d425ed4d44cb6e886dc769\">https://github.com/django/django/commit/1abcf3a808b35abae5d425ed4d44cb6e886dc769</a></li>\n<li><a href=\"https://github.com/django/django/commit/28e23306aa53bbbb8fb87db85f99d970b051026c\">https://github.com/django/django/commit/28e23306aa53bbbb8fb87db85f99d970b051026c</a></li>\n<li><a href=\"https://github.com/django/django/commit/4001ec8698f577b973c5a540801d8a0bbea1205b\">https://github.com/django/django/commit/4001ec8698f577b973c5a540801d8a0bbea1205b</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued\">https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2014-19.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2014-19.yaml</a></li>\n<li><a href=\"http://ubuntu.com/usn/usn-2212-1\">http://ubuntu.com/usn/usn-2212-1</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-q7q2-qf2q-rw3w\">https://github.com/advisories/GHSA-q7q2-qf2q-rw3w</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-q7q2-qf2q-rw3w",
            "title": "[Django] Django Vulnerable to Cache Poisoning",
            "date_modified": "2024-09-18T19:51:16.000Z",
            "date_published": "2022-05-17T03:07:00.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-q7q2-qf2q-rw3w"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Django 1.4 before 1.4.13, 1.5 before 1.5.8, 1.6 before 1.6.5, and 1.7 before 1.7b4 does not properly include the (1) Vary: Cookie or (2) Cache-Control header in responses, which allows remote attackers to obtain sensitive information or poison the cache via a request from certain browsers.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2014-1418\">https://nvd.nist.gov/vuln/detail/CVE-2014-1418</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html\">http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html</a></li>\n<li><a href=\"http://www.debian.org/security/2014/dsa-2934\">http://www.debian.org/security/2014/dsa-2934</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2014/05/14/10\">http://www.openwall.com/lists/oss-security/2014/05/14/10</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2014/05/15/3\">http://www.openwall.com/lists/oss-security/2014/05/15/3</a></li>\n<li><a href=\"https://github.com/django/django/commit/1abcf3a808b35abae5d425ed4d44cb6e886dc769\">https://github.com/django/django/commit/1abcf3a808b35abae5d425ed4d44cb6e886dc769</a></li>\n<li><a href=\"https://github.com/django/django/commit/28e23306aa53bbbb8fb87db85f99d970b051026c\">https://github.com/django/django/commit/28e23306aa53bbbb8fb87db85f99d970b051026c</a></li>\n<li><a href=\"https://github.com/django/django/commit/4001ec8698f577b973c5a540801d8a0bbea1205b\">https://github.com/django/django/commit/4001ec8698f577b973c5a540801d8a0bbea1205b</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued\">https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2014-19.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2014-19.yaml</a></li>\n<li><a href=\"http://ubuntu.com/usn/usn-2212-1\">http://ubuntu.com/usn/usn-2212-1</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-q7q2-qf2q-rw3w\">https://github.com/advisories/GHSA-q7q2-qf2q-rw3w</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-q7q2-qf2q-rw3w",
            "title": "[Django] Django Vulnerable to Cache Poisoning",
            "date_modified": "2024-09-18T19:51:16.000Z",
            "date_published": "2022-05-17T03:07:00.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-q7q2-qf2q-rw3w"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Django 1.4 before 1.4.13, 1.5 before 1.5.8, 1.6 before 1.6.5, and 1.7 before 1.7b4 does not properly include the (1) Vary: Cookie or (2) Cache-Control header in responses, which allows remote attackers to obtain sensitive information or poison the cache via a request from certain browsers.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2014-1418\">https://nvd.nist.gov/vuln/detail/CVE-2014-1418</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html\">http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html</a></li>\n<li><a href=\"http://www.debian.org/security/2014/dsa-2934\">http://www.debian.org/security/2014/dsa-2934</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2014/05/14/10\">http://www.openwall.com/lists/oss-security/2014/05/14/10</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2014/05/15/3\">http://www.openwall.com/lists/oss-security/2014/05/15/3</a></li>\n<li><a href=\"https://github.com/django/django/commit/1abcf3a808b35abae5d425ed4d44cb6e886dc769\">https://github.com/django/django/commit/1abcf3a808b35abae5d425ed4d44cb6e886dc769</a></li>\n<li><a href=\"https://github.com/django/django/commit/28e23306aa53bbbb8fb87db85f99d970b051026c\">https://github.com/django/django/commit/28e23306aa53bbbb8fb87db85f99d970b051026c</a></li>\n<li><a href=\"https://github.com/django/django/commit/4001ec8698f577b973c5a540801d8a0bbea1205b\">https://github.com/django/django/commit/4001ec8698f577b973c5a540801d8a0bbea1205b</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued\">https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2014-19.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2014-19.yaml</a></li>\n<li><a href=\"http://ubuntu.com/usn/usn-2212-1\">http://ubuntu.com/usn/usn-2212-1</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-q7q2-qf2q-rw3w\">https://github.com/advisories/GHSA-q7q2-qf2q-rw3w</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-q7q2-qf2q-rw3w",
            "title": "[Django] Django Vulnerable to Cache Poisoning",
            "date_modified": "2024-09-18T19:51:16.000Z",
            "date_published": "2022-05-17T03:07:00.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-q7q2-qf2q-rw3w"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Cross-site scripting (XSS) vulnerability in the AdminURLFieldWidget widget in contrib/admin/widgets.py in Django 1.5.x before 1.5.2 and 1.6.x before 1.6 beta 2 allows remote attackers to inject arbitrary web script or HTML via a URLField.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2013-4249\">https://nvd.nist.gov/vuln/detail/CVE-2013-4249</a></li>\n<li><a href=\"https://github.com/django/django/commit/90363e388c61874add3f3557ee654a996ec75d78\">https://github.com/django/django/commit/90363e388c61874add3f3557ee654a996ec75d78</a></li>\n<li><a href=\"https://github.com/django/django/commit/cbe6d5568f4f5053ed7228ca3c3d0cce77cf9560\">https://github.com/django/django/commit/cbe6d5568f4f5053ed7228ca3c3d0cce77cf9560</a></li>\n<li><a href=\"https://exchange.xforce.ibmcloud.com/vulnerabilities/86438\">https://exchange.xforce.ibmcloud.com/vulnerabilities/86438</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2013/aug/13/security-releases-issued\">https://www.djangoproject.com/weblog/2013/aug/13/security-releases-issued</a></li>\n<li><a href=\"http://seclists.org/oss-sec/2013/q3/369\">http://seclists.org/oss-sec/2013/q3/369</a></li>\n<li><a href=\"http://seclists.org/oss-sec/2013/q3/411\">http://seclists.org/oss-sec/2013/q3/411</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2013-19.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2013-19.yaml</a></li>\n<li><a href=\"https://web.archive.org/web/20201208180405/http://www.securitytracker.com/id/1028915\">https://web.archive.org/web/20201208180405/http://www.securitytracker.com/id/1028915</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-4894-5vqc-6r2r\">https://github.com/advisories/GHSA-4894-5vqc-6r2r</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-4894-5vqc-6r2r",
            "title": "[django] Django cross-site scripting (XSS) vulnerability in the AdminURLFieldWidget widget",
            "date_modified": "2024-09-18T19:49:06.000Z",
            "date_published": "2022-05-17T01:33:07.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-4894-5vqc-6r2r"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p><code>django.contrib.auth.forms.AuthenticationForm</code> in Django 2.0 before 2.0.2, and 1.11.8 and 1.11.9, allows remote attackers to obtain potentially sensitive information by leveraging data exposure from the <code>confirm_login_allowed()</code> method, as demonstrated by discovering whether a user account is inactive.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2018-6188\">https://nvd.nist.gov/vuln/detail/CVE-2018-6188</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-rf4j-j272-fj86\">https://github.com/advisories/GHSA-rf4j-j272-fj86</a></li>\n<li><a href=\"https://github.com/django/django/commit/57b95fedad5e0b83fc9c81466b7d1751c6427aae\">https://github.com/django/django/commit/57b95fedad5e0b83fc9c81466b7d1751c6427aae</a></li>\n<li><a href=\"https://github.com/django/django/commit/c37bb28677295f6edda61d8ac461014ef0d3aeb2\">https://github.com/django/django/commit/c37bb28677295f6edda61d8ac461014ef0d3aeb2</a></li>\n<li><a href=\"https://usn.ubuntu.com/3559-1\">https://usn.ubuntu.com/3559-1</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2018/feb/01/security-releases\">https://www.djangoproject.com/weblog/2018/feb/01/security-releases</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2018-4.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2018-4.yaml</a></li>\n<li><a href=\"https://web.archive.org/web/20200517143909/http://www.securitytracker.com/id/1040422\">https://web.archive.org/web/20200517143909/http://www.securitytracker.com/id/1040422</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-rf4j-j272-fj86",
            "title": "[Django] Django vulnerable to information leakage in AuthenticationForm",
            "date_modified": "2024-09-18T19:46:35.000Z",
            "date_published": "2018-10-03T21:13:54.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-rf4j-j272-fj86"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>An issue was discovered in Django 2.0 before 2.0.3, 1.11 before 1.11.11, and 1.8 before 1.8.19. The <code>django.utils.html.urlize()</code> function was extremely slow to evaluate certain inputs due to catastrophic backtracking vulnerabilities in two regular expressions (only one regular expression for Django 1.8.x). The <code>urlize()</code> function is used to implement the urlize and urlizetrunc template filters, which were thus vulnerable.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2018-7536\">https://nvd.nist.gov/vuln/detail/CVE-2018-7536</a></li>\n<li><a href=\"https://access.redhat.com/errata/RHSA-2018:2927\">https://access.redhat.com/errata/RHSA-2018:2927</a></li>\n<li><a href=\"https://access.redhat.com/errata/RHSA-2019:0051\">https://access.redhat.com/errata/RHSA-2019:0051</a></li>\n<li><a href=\"https://access.redhat.com/errata/RHSA-2019:0082\">https://access.redhat.com/errata/RHSA-2019:0082</a></li>\n<li><a href=\"https://access.redhat.com/errata/RHSA-2019:0265\">https://access.redhat.com/errata/RHSA-2019:0265</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-r28v-mw67-m5p9\">https://github.com/advisories/GHSA-r28v-mw67-m5p9</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2018/03/msg00006.html\">https://lists.debian.org/debian-lts-announce/2018/03/msg00006.html</a></li>\n<li><a href=\"https://www.debian.org/security/2018/dsa-4161\">https://www.debian.org/security/2018/dsa-4161</a></li>\n<li><a href=\"https://github.com/django/django/commit/1ca63a66ef3163149ad822701273e8a1844192c2\">https://github.com/django/django/commit/1ca63a66ef3163149ad822701273e8a1844192c2</a></li>\n<li><a href=\"https://github.com/django/django/commit/abf89d729f210c692a50e0ad3f75fb6bec6fae16\">https://github.com/django/django/commit/abf89d729f210c692a50e0ad3f75fb6bec6fae16</a></li>\n<li><a href=\"https://github.com/django/django/commit/e157315da3ae7005fa0683ffc9751dbeca7306c8\">https://github.com/django/django/commit/e157315da3ae7005fa0683ffc9751dbeca7306c8</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2018-5.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2018-5.yaml</a></li>\n<li><a href=\"https://usn.ubuntu.com/3591-1\">https://usn.ubuntu.com/3591-1</a></li>\n<li><a href=\"https://web.archive.org/web/20200227131019/http://www.securityfocus.com/bid/103361\">https://web.archive.org/web/20200227131019/http://www.securityfocus.com/bid/103361</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2018/mar/06/security-releases\">https://www.djangoproject.com/weblog/2018/mar/06/security-releases</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-r28v-mw67-m5p9",
            "title": "[Django] Django denial-of-service possibility in urlize and urlizetrunc template filters",
            "date_modified": "2024-09-18T19:45:20.000Z",
            "date_published": "2019-01-04T17:50:07.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-r28v-mw67-m5p9"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>An issue was discovered in Django 2.0 before 2.0.3, 1.11 before 1.11.11, and 1.8 before 1.8.19. The <code>django.utils.html.urlize()</code> function was extremely slow to evaluate certain inputs due to catastrophic backtracking vulnerabilities in two regular expressions (only one regular expression for Django 1.8.x). The <code>urlize()</code> function is used to implement the urlize and urlizetrunc template filters, which were thus vulnerable.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2018-7536\">https://nvd.nist.gov/vuln/detail/CVE-2018-7536</a></li>\n<li><a href=\"https://access.redhat.com/errata/RHSA-2018:2927\">https://access.redhat.com/errata/RHSA-2018:2927</a></li>\n<li><a href=\"https://access.redhat.com/errata/RHSA-2019:0051\">https://access.redhat.com/errata/RHSA-2019:0051</a></li>\n<li><a href=\"https://access.redhat.com/errata/RHSA-2019:0082\">https://access.redhat.com/errata/RHSA-2019:0082</a></li>\n<li><a href=\"https://access.redhat.com/errata/RHSA-2019:0265\">https://access.redhat.com/errata/RHSA-2019:0265</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-r28v-mw67-m5p9\">https://github.com/advisories/GHSA-r28v-mw67-m5p9</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2018/03/msg00006.html\">https://lists.debian.org/debian-lts-announce/2018/03/msg00006.html</a></li>\n<li><a href=\"https://www.debian.org/security/2018/dsa-4161\">https://www.debian.org/security/2018/dsa-4161</a></li>\n<li><a href=\"https://github.com/django/django/commit/1ca63a66ef3163149ad822701273e8a1844192c2\">https://github.com/django/django/commit/1ca63a66ef3163149ad822701273e8a1844192c2</a></li>\n<li><a href=\"https://github.com/django/django/commit/abf89d729f210c692a50e0ad3f75fb6bec6fae16\">https://github.com/django/django/commit/abf89d729f210c692a50e0ad3f75fb6bec6fae16</a></li>\n<li><a href=\"https://github.com/django/django/commit/e157315da3ae7005fa0683ffc9751dbeca7306c8\">https://github.com/django/django/commit/e157315da3ae7005fa0683ffc9751dbeca7306c8</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2018-5.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2018-5.yaml</a></li>\n<li><a href=\"https://usn.ubuntu.com/3591-1\">https://usn.ubuntu.com/3591-1</a></li>\n<li><a href=\"https://web.archive.org/web/20200227131019/http://www.securityfocus.com/bid/103361\">https://web.archive.org/web/20200227131019/http://www.securityfocus.com/bid/103361</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2018/mar/06/security-releases\">https://www.djangoproject.com/weblog/2018/mar/06/security-releases</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-r28v-mw67-m5p9",
            "title": "[Django] Django denial-of-service possibility in urlize and urlizetrunc template filters",
            "date_modified": "2024-09-18T19:45:20.000Z",
            "date_published": "2019-01-04T17:50:07.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-r28v-mw67-m5p9"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>An issue was discovered in Django 2.0 before 2.0.3, 1.11 before 1.11.11, and 1.8 before 1.8.19. The <code>django.utils.html.urlize()</code> function was extremely slow to evaluate certain inputs due to catastrophic backtracking vulnerabilities in two regular expressions (only one regular expression for Django 1.8.x). The <code>urlize()</code> function is used to implement the urlize and urlizetrunc template filters, which were thus vulnerable.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2018-7536\">https://nvd.nist.gov/vuln/detail/CVE-2018-7536</a></li>\n<li><a href=\"https://access.redhat.com/errata/RHSA-2018:2927\">https://access.redhat.com/errata/RHSA-2018:2927</a></li>\n<li><a href=\"https://access.redhat.com/errata/RHSA-2019:0051\">https://access.redhat.com/errata/RHSA-2019:0051</a></li>\n<li><a href=\"https://access.redhat.com/errata/RHSA-2019:0082\">https://access.redhat.com/errata/RHSA-2019:0082</a></li>\n<li><a href=\"https://access.redhat.com/errata/RHSA-2019:0265\">https://access.redhat.com/errata/RHSA-2019:0265</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-r28v-mw67-m5p9\">https://github.com/advisories/GHSA-r28v-mw67-m5p9</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2018/03/msg00006.html\">https://lists.debian.org/debian-lts-announce/2018/03/msg00006.html</a></li>\n<li><a href=\"https://www.debian.org/security/2018/dsa-4161\">https://www.debian.org/security/2018/dsa-4161</a></li>\n<li><a href=\"https://github.com/django/django/commit/1ca63a66ef3163149ad822701273e8a1844192c2\">https://github.com/django/django/commit/1ca63a66ef3163149ad822701273e8a1844192c2</a></li>\n<li><a href=\"https://github.com/django/django/commit/abf89d729f210c692a50e0ad3f75fb6bec6fae16\">https://github.com/django/django/commit/abf89d729f210c692a50e0ad3f75fb6bec6fae16</a></li>\n<li><a href=\"https://github.com/django/django/commit/e157315da3ae7005fa0683ffc9751dbeca7306c8\">https://github.com/django/django/commit/e157315da3ae7005fa0683ffc9751dbeca7306c8</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2018-5.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2018-5.yaml</a></li>\n<li><a href=\"https://usn.ubuntu.com/3591-1\">https://usn.ubuntu.com/3591-1</a></li>\n<li><a href=\"https://web.archive.org/web/20200227131019/http://www.securityfocus.com/bid/103361\">https://web.archive.org/web/20200227131019/http://www.securityfocus.com/bid/103361</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2018/mar/06/security-releases\">https://www.djangoproject.com/weblog/2018/mar/06/security-releases</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-r28v-mw67-m5p9",
            "title": "[Django] Django denial-of-service possibility in urlize and urlizetrunc template filters",
            "date_modified": "2024-09-18T19:45:20.000Z",
            "date_published": "2019-01-04T17:50:07.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-r28v-mw67-m5p9"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>The <code>django.util.http.is_safe_url</code> function in Django 1.4 before 1.4.13, 1.5 before 1.5.8, 1.6 before 1.6.5, and 1.7 before 1.7b4 does not properly validate URLs, which allows remote attackers to conduct open redirect attacks via a malformed URL, as demonstrated by \"http:\\\\djangoproject.com.\"</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2014-3730\">https://nvd.nist.gov/vuln/detail/CVE-2014-3730</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html\">http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html</a></li>\n<li><a href=\"http://ubuntu.com/usn/usn-2212-1\">http://ubuntu.com/usn/usn-2212-1</a></li>\n<li><a href=\"http://www.debian.org/security/2014/dsa-2934\">http://www.debian.org/security/2014/dsa-2934</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2014/05/14/10\">http://www.openwall.com/lists/oss-security/2014/05/14/10</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2014/05/15/3\">http://www.openwall.com/lists/oss-security/2014/05/15/3</a></li>\n<li><a href=\"https://github.com/django/django/commit/601107524523bca02376a0ddc1a06c6fdb8f22f3\">https://github.com/django/django/commit/601107524523bca02376a0ddc1a06c6fdb8f22f3</a></li>\n<li><a href=\"https://github.com/django/django/commit/7feb54bbae3f637ab3c4dd4831d4385964f574df\">https://github.com/django/django/commit/7feb54bbae3f637ab3c4dd4831d4385964f574df</a></li>\n<li><a href=\"https://github.com/django/django/commit/ad32c218850ad40972dcef57beb460f8c979dd6d\">https://github.com/django/django/commit/ad32c218850ad40972dcef57beb460f8c979dd6d</a></li>\n<li><a href=\"https://web.archive.org/web/20200228171223/http://www.securityfocus.com/bid/67410\">https://web.archive.org/web/20200228171223/http://www.securityfocus.com/bid/67410</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2014-20.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2014-20.yaml</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued\">https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued</a></li>\n<li><a href=\"http://secunia.com/advisories/61281\">http://secunia.com/advisories/61281</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-vq3h-3q7v-9prw\">https://github.com/advisories/GHSA-vq3h-3q7v-9prw</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-vq3h-3q7v-9prw",
            "title": "[Django] Django Allows Open Redirects",
            "date_modified": "2024-09-18T19:43:08.000Z",
            "date_published": "2022-05-14T02:09:43.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-vq3h-3q7v-9prw"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>The <code>django.util.http.is_safe_url</code> function in Django 1.4 before 1.4.13, 1.5 before 1.5.8, 1.6 before 1.6.5, and 1.7 before 1.7b4 does not properly validate URLs, which allows remote attackers to conduct open redirect attacks via a malformed URL, as demonstrated by \"http:\\\\djangoproject.com.\"</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2014-3730\">https://nvd.nist.gov/vuln/detail/CVE-2014-3730</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html\">http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html</a></li>\n<li><a href=\"http://ubuntu.com/usn/usn-2212-1\">http://ubuntu.com/usn/usn-2212-1</a></li>\n<li><a href=\"http://www.debian.org/security/2014/dsa-2934\">http://www.debian.org/security/2014/dsa-2934</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2014/05/14/10\">http://www.openwall.com/lists/oss-security/2014/05/14/10</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2014/05/15/3\">http://www.openwall.com/lists/oss-security/2014/05/15/3</a></li>\n<li><a href=\"https://github.com/django/django/commit/601107524523bca02376a0ddc1a06c6fdb8f22f3\">https://github.com/django/django/commit/601107524523bca02376a0ddc1a06c6fdb8f22f3</a></li>\n<li><a href=\"https://github.com/django/django/commit/7feb54bbae3f637ab3c4dd4831d4385964f574df\">https://github.com/django/django/commit/7feb54bbae3f637ab3c4dd4831d4385964f574df</a></li>\n<li><a href=\"https://github.com/django/django/commit/ad32c218850ad40972dcef57beb460f8c979dd6d\">https://github.com/django/django/commit/ad32c218850ad40972dcef57beb460f8c979dd6d</a></li>\n<li><a href=\"https://web.archive.org/web/20200228171223/http://www.securityfocus.com/bid/67410\">https://web.archive.org/web/20200228171223/http://www.securityfocus.com/bid/67410</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2014-20.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2014-20.yaml</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued\">https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued</a></li>\n<li><a href=\"http://secunia.com/advisories/61281\">http://secunia.com/advisories/61281</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-vq3h-3q7v-9prw\">https://github.com/advisories/GHSA-vq3h-3q7v-9prw</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-vq3h-3q7v-9prw",
            "title": "[Django] Django Allows Open Redirects",
            "date_modified": "2024-09-18T19:43:08.000Z",
            "date_published": "2022-05-14T02:09:43.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-vq3h-3q7v-9prw"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>The <code>django.util.http.is_safe_url</code> function in Django 1.4 before 1.4.13, 1.5 before 1.5.8, 1.6 before 1.6.5, and 1.7 before 1.7b4 does not properly validate URLs, which allows remote attackers to conduct open redirect attacks via a malformed URL, as demonstrated by \"http:\\\\djangoproject.com.\"</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2014-3730\">https://nvd.nist.gov/vuln/detail/CVE-2014-3730</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html\">http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html</a></li>\n<li><a href=\"http://ubuntu.com/usn/usn-2212-1\">http://ubuntu.com/usn/usn-2212-1</a></li>\n<li><a href=\"http://www.debian.org/security/2014/dsa-2934\">http://www.debian.org/security/2014/dsa-2934</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2014/05/14/10\">http://www.openwall.com/lists/oss-security/2014/05/14/10</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2014/05/15/3\">http://www.openwall.com/lists/oss-security/2014/05/15/3</a></li>\n<li><a href=\"https://github.com/django/django/commit/601107524523bca02376a0ddc1a06c6fdb8f22f3\">https://github.com/django/django/commit/601107524523bca02376a0ddc1a06c6fdb8f22f3</a></li>\n<li><a href=\"https://github.com/django/django/commit/7feb54bbae3f637ab3c4dd4831d4385964f574df\">https://github.com/django/django/commit/7feb54bbae3f637ab3c4dd4831d4385964f574df</a></li>\n<li><a href=\"https://github.com/django/django/commit/ad32c218850ad40972dcef57beb460f8c979dd6d\">https://github.com/django/django/commit/ad32c218850ad40972dcef57beb460f8c979dd6d</a></li>\n<li><a href=\"https://web.archive.org/web/20200228171223/http://www.securityfocus.com/bid/67410\">https://web.archive.org/web/20200228171223/http://www.securityfocus.com/bid/67410</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2014-20.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2014-20.yaml</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued\">https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued</a></li>\n<li><a href=\"http://secunia.com/advisories/61281\">http://secunia.com/advisories/61281</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-vq3h-3q7v-9prw\">https://github.com/advisories/GHSA-vq3h-3q7v-9prw</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-vq3h-3q7v-9prw",
            "title": "[Django] Django Allows Open Redirects",
            "date_modified": "2024-09-18T19:43:08.000Z",
            "date_published": "2022-05-14T02:09:43.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-vq3h-3q7v-9prw"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>The <code>django.util.http.is_safe_url</code> function in Django 1.4 before 1.4.13, 1.5 before 1.5.8, 1.6 before 1.6.5, and 1.7 before 1.7b4 does not properly validate URLs, which allows remote attackers to conduct open redirect attacks via a malformed URL, as demonstrated by \"http:\\\\djangoproject.com.\"</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2014-3730\">https://nvd.nist.gov/vuln/detail/CVE-2014-3730</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html\">http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html</a></li>\n<li><a href=\"http://ubuntu.com/usn/usn-2212-1\">http://ubuntu.com/usn/usn-2212-1</a></li>\n<li><a href=\"http://www.debian.org/security/2014/dsa-2934\">http://www.debian.org/security/2014/dsa-2934</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2014/05/14/10\">http://www.openwall.com/lists/oss-security/2014/05/14/10</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2014/05/15/3\">http://www.openwall.com/lists/oss-security/2014/05/15/3</a></li>\n<li><a href=\"https://github.com/django/django/commit/601107524523bca02376a0ddc1a06c6fdb8f22f3\">https://github.com/django/django/commit/601107524523bca02376a0ddc1a06c6fdb8f22f3</a></li>\n<li><a href=\"https://github.com/django/django/commit/7feb54bbae3f637ab3c4dd4831d4385964f574df\">https://github.com/django/django/commit/7feb54bbae3f637ab3c4dd4831d4385964f574df</a></li>\n<li><a href=\"https://github.com/django/django/commit/ad32c218850ad40972dcef57beb460f8c979dd6d\">https://github.com/django/django/commit/ad32c218850ad40972dcef57beb460f8c979dd6d</a></li>\n<li><a href=\"https://web.archive.org/web/20200228171223/http://www.securityfocus.com/bid/67410\">https://web.archive.org/web/20200228171223/http://www.securityfocus.com/bid/67410</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2014-20.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2014-20.yaml</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued\">https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued</a></li>\n<li><a href=\"http://secunia.com/advisories/61281\">http://secunia.com/advisories/61281</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-vq3h-3q7v-9prw\">https://github.com/advisories/GHSA-vq3h-3q7v-9prw</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-vq3h-3q7v-9prw",
            "title": "[Django] Django Allows Open Redirects",
            "date_modified": "2024-09-18T19:43:08.000Z",
            "date_published": "2022-05-14T02:09:43.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-vq3h-3q7v-9prw"
            },
            "tags": [
                "severity"
            ]
        }
    ]
}